README.md

```diff
@@ -17,11 +17,6 @@
 - GitFlow
 - Webpack
 
-[Live Demo Link](https://livedemo.com)
-## Live Demo
-
-[Live Demo Link coming soon...](#)
-
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
```

Comment:
- [ ] Please remove these lines from the README file, This is just an example of how you can give instructions, instead It will be nice to add explicit steps on how to clone your application as it will be helpful to non-technical users. This includes `git clone URL` , `cd project_name`, etc 🙃